eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -121,16 +121,41 @@
     <!-- Known issue in autorest where it will generate a variable with a value of null then use it in a not null check.
     https://github.com/Azure/autorest.java/issues/328 -->
     <Match>
-        <Class name="~com\.microsoft\.azure\.batch\.protocol\.implementation\.[\w]+Impl"/>
-        <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE"/>
+      <Class name="~com\.microsoft\.azure\.batch\.protocol\.implementation\.[\w]+Impl"/>
+      <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE"/>
     </Match>
 
     <Match>
-        <Class name="com.microsoft.azure.batch.CreateTasksErrorException"/>
-        <Or>
-            <Field name="failureTaskList"/>
-            <Field name="pendingTaskList"/>
-        </Or>
-        <Bug pattern="SE_BAD_FIELD"/>
+      <Class name="com.microsoft.azure.batch.CreateTasksErrorException"/>
+      <Or>
+        <Field name="failureTaskList"/>
+        <Field name="pendingTaskList"/>
+      </Or>
+      <Bug pattern="SE_BAD_FIELD"/>
+    </Match>
+
+    <!-- Methods in Common package return null instead of meaningful instances when resourses do not exist. Will address the bugs in Track Two.
+    -->
+    <Match>
+      <Class name="~com\.azure\.common\.implementation\.(Base64Url|serializer\.HttpResponseDecodeData|util\.ImplUtils)"/>
+      <Bug pattern="PZLA_PREFER_ZERO_LENGTH_ARRAYS"/>
+    </Match>
```

Comment:
You no longer need this match statement, do you?

Reply:
Base64Url, SwaggerMethodParser, HttpResponseDecodeData, are these three files used by anyone else? If no one uses them, I can fix the bugs instead of suppressing them

Reply:
Yes, fix these bugs rather than suppressing. Do it in conjunction with @anuchandy as he knows this code well! :-)

Reply:
I actually tried to fix the bugs. In ImplUtils class, I leave two methods return null in case some of the packages cannot change it zero length array when fix the spotbugs.

Reply:
suppress/non-suppress - we need to decide based on the conclusion of this active [discussion](https://github.com/Azure/azure-sdk-for-java/pull/3284#discussion_r273081891)